src/main/java/seedu/address/model/pet/Appointment.java

```diff
@@ -79,6 +80,10 @@ public String getLocation() {
         return this.location;
     }
 
+    public LocalDate getDateOnly() {
```

Comment:
Missing Javadoc comment here!

Reply:
Thanks good catch! I realised that I did not use the method so I am going to delete it for now.